Source/Core scripts/MarchingCubesView.cs

```diff
@@ -150,6 +150,8 @@ public bool InvertedNormals
 
                 invertedNormals = value;
             }
+
+            get => invertedNormals;
```

Comment:
I think it would make more sense to store this value in the controller instead of getting it from element 0. Since the controller should be responsible for this value.
If for example the view is destroyed and recreated because of another process, this value might be set to whatever default the view has.

Reply:
I agree, will add it to the controller.